### ISSUE TITLE
README on supervisord.conf issue

### ISSUE THREAD

**Issue by ryanjmccall** [oxtopus/skeleton-htmengine-app#12]
This repo's README suggests that the [supervisord] "environment" property points to the htmengine conf dir. However, this example (https://github.com/htm-community/htmengine-traffic-tutorial/blob/master/python-engine/conf/supervisord.conf) points "environment" at the conf dir of the user's custom "skeleton" project. Additionally the [include] "files" property instead points points to the supervisor-base.conf in htmengine. @oxtopus

**Comment by oxtopus** [comment]
Yes, you're right.  For that matter, the value should match the `APPLICATION_CONFIG_PATH` value described in section 3: https://github.com/htm-community/skeleton-htmengine-app/blob/master/README.md#3-set-application_config_path-in-your-environment

I believe the _include_ directive should point to `htmengine` conf as is described in the readme.
